test/SILGen/switch.swift

```diff
@@ -1098,3 +1098,21 @@ func testOptionalEnumMixWithNil(_ a : Int?) -> Int {
   // CHECK: integer_literal $Builtin.Int2048, 42
   }
 }
+
+// SR-3518
+func testMultiPatternsWithOuterScopeSameNamedVar(base: Int?, filter: Int?) {
```

Comment:
Add a CHECK-LABEL: for the function itself to ensure we don't pick up random SIL instructions elsewhere in the file